.gitignore

```diff
@@ -1,3 +1,2 @@
 node_modules
 test/sandbox
-test/sandbox2
```

Comment:
Appending a number to directory name does not scale well. If you have a test that needs multiple directories, then you should use nested subdirectories inside `test/sandbox`, e.g. `test/sandbox/app1` and `test/sandbox/app2`.